R/lastdose.R

```diff
@@ -185,7 +183,7 @@ lastdose_list <- function(data,
   if(!is.numeric(col_addl)) {
     stop("column ADDL/addl is required to be numeric", call.=FALSE)
   }
-  wii <- match("ii", na)
```

Comment:
This is the only offending code; the other changes were to factor out using `na`